database/01_table_creation.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE token_info (
+    token_address bytea PRIMARY KEY,
+    symbol varchar NOT NULL,
```

Comment:
Tbh the symbol is not needed, so i would remove it

Reply:
Then lets just remove it. No need to store unused data here.